src/main/kotlin/skytils/skytilsmod/core/Config.kt

```diff
@@ -1572,6 +1572,14 @@ object Config : Vigilant(File("./config/skytils/config.toml"), "Skytils", sortin
     )
     var powerOrbLock = false
 
+    @Property(
+        type = PropertyType.NUMBER, name = "Power Orb Lock Duration",
+        description = "Needs Power Orb Lock to be active. Allows overwriting a power orb, if it has less time left than this option.",
+        min = 1, max = 60,
```

Comment:
the bubblegum pet item doubles the time but idk if anyone cares

Reply:
alright, should i just preemptively remove the max, expecting future updates with even more powerful balls / pet items, or upgrade to 120 secs

Reply:
I think remove the max but idk if vigilance will work with that

Reply:
It does not work with just one limit. Ig ill just put 120 as limit for now

Reply:
Default vigilance max is 100 and it'll be that if you don't specify one

Reply:
Actually it was 0 in my tests

Reply:
I think 100 is the default max for PERCENTAGE_SLIDER